restcomm/restcomm.docs/sources-asciidoc/src/main/asciidoc/configuration/Starting Restcomm-Connect.adoc

```diff
@@ -38,21 +47,33 @@ image::images/10.png[]
 
 * Configure the following variables with the network configuration details of your server:
 
-image::images/9.png[]
+[source,bash]
+----
+# Network configuration
+NET_INTERFACE='wlx60e32716068a'
+PRIVATE_IP='172.21.0.107'
+SUBNET_MASK='255.255.255.0'
+NETWORK='172.21.0.0'
+BROADCAST_ADDRESS='172.21.0.255'
+----
 
-*Add your Text-to-Speech Key*
+=== Add your Text-to-Speech Key
 
```

Comment:
We should probably add a paragraph on why this is required and what the Text to Speech is used for for developers not use to telecommunications